phase2/src/utils.rs

```diff
@@ -63,7 +63,7 @@ pub fn merge_pairs<G: CurveAffine>(v1: &[G], v2: &[G]) -> (G, G)
 
     assert_eq!(v1.len(), v2.len());
 
-    let chunk = (v1.len() / num_cpus::get()) + 1;
+    let chunk = get_chunk_size(v1.len());
```

Comment:
Note that the old implementation sets `chunk` to be ⌊v1.len() / num_cpus⌋ + 1 but the `get_chunk_size` method in the new implementation returns ⌈v1.len() / num_cpus⌉, assuming that `v1.len()` is non-zero.

Reply:
This is intentional. The original implementation is not optimal.